linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_ocredit/tests/ospp.pass.sh

```diff
@@ -1,5 +1,4 @@
 #!/bin/bash
 #
-# profiles = xccdf_org.ssgproject.content_profile_ospp
```

Comment:
There is variable `var_password_pam_ocredit.var` involved in this and for some profiles, this shouldn't pass, eg. rht-ccp profile requires ocredit to be set to -2.